xml/deployment_installing_nodes.xml

```diff
@@ -1090,4 +1090,43 @@
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="sec.deploy.install.vmware_tools">
+  <title>Installing &vmware; Tools</title>
+  <para>
+   This section is only relevant for deployments on &vmware; ESX and
+   ESXi environments. This step is not required if you are using
+   virtual disk images as described in <xref
+   linkend="sec.deploy.preparation.disk_images" />, because
+   <package>open-vm-tools</package> is already installed.
+  </para>
+  <para>
+   After the bootstrapping of the cluster is finished, install the
+   &vmware; tools on all nodes that are included in the package
+   <package>open-vm-tools</package>. Log in on the &admin_node; and
+   execute:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Install <package>open-vm-tools</package> on all nodes.
+    </para>
+<screen>&prompt.root.admin;<command>docker exec $(docker ps -q --filter name=salt-master) salt -P \
```

Comment:
Maybe we should choose one the method for printing out salt-master container id:
* $(docker ps -q --filter name=salt-master)
* $(docker ps | grep salt-master | awk '{print $1}') ... at line 1122

Both methods are equivalent - I would prefer pure docker way and not docker+grep+awk

Reply:
We use the grep+awk method for most of the other commands. If you do change this to the first version, please let me know so we can check where else this is used.

Reply:
@r0ckarong I agree with @thehejik on this point. Do you have an opinion here? I like the suggestion and would replace all grep-awk-based commands with the Salt-only solution.

Reply:
I agree that this is the better way to do it. It just returns the necessary info other than producing all the info and cutting it down.
I would want to replace this with: `$(docker ps -q -f name salt-master)` (even shorter)
or equivalents across the guides. Please change it across this PR and I'll adjust it in develop and check my other PRs.